lib/proto.js

```diff
@@ -85,6 +86,19 @@ function child (bindings) {
   return instance
 }
 
+function childBindings () {
+  const chindings = this[chindingsSym]
+  var chindingsObject = (chindings.indexOf(',') === 0) ? `{${chindings.substr(1)}}` : `{${chindings}}`
+  try {
+    var bindings = JSON.parse(chindingsObject) || {}
+    delete bindings['pid']
```

Comment:
avoid square bracket notation unless require - e.g. use `bindings.pid`  (and same below)